bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/ApiBridgeHandler.java

```diff
@@ -181,10 +182,6 @@ public void openConnection(@Nullable String code, @Nullable String redirectUri)
         }
 
         updateStatus(ThingStatus.ONLINE);
-
-        getThing().getThings().stream().filter(Thing::isEnabled).map(Thing::getHandler)
-                .filter(CommonInterface.class::isInstance).map(CommonInterface.class::cast)
-                .forEach(CommonInterface::expireData);
```

Comment:
No more required ?

Reply:
No, during the tests I made, I figured that `updateStatus(ThingStatus.ONLINE);` already trigger a general `expireData`. I've experimented many times and it did not appear to add anything.

Reply:
Hope you're right